Packs/LinuxEventsCollection/README.md

```diff
@@ -1,52 +1,75 @@
 # Linux Events Collection
-This pack includes XSIAM content.
 
-## Collect Events from Vendor
+## Collector Configuration
 
-In order to use the collector, you can use one of the following options to collect events from the vendor:
+You could use any one of the following options to collect Linux events into Cortex XSIAM:
+  - [Broker VM Syslog Applet](#broker-vm)
+  - [Filebeat XDRC (XDR Collector)](#xdrc-xdr-collector)
 
-- [Linux Events Collection](#linux-events-collection)
-  - [Collect Events from Vendor](#collect-events-from-vendor)
-    - [Broker VM](#broker-vm)
-    - [XDRC (XDR Collector)](#xdrc-xdr-collector)
-
-In either option, you will need to configure the vendor and product for this specific collector.
+After the Cortex XSIAM Collector starts ingesting logs from the configured paths on the Linux servers, you could query the collected logs under the *`linux_linux_raw`* dataset. 
 
 ### Broker VM
 
-You will need to use the information described [here](https://docs-cortex.paloaltonetworks.com/r/Cortex-XDR/Cortex-XDR-Pro-Administrator-Guide/Configure-the-Broker-VM).\
-You can configure the specific vendor and product for this instance.
+#### Prerequisites
 
-1. Navigate to **Settings** -> **Configuration** -> **Data Broker** -> **Broker VMs**.
-2. Right-click, and select **Syslog Collector** -> **Configure**.
-3. When configuring the Syslog Collector, set:
-   - vendor as linux
-   - product as linux
+You would need to have a Broker VM set and configured as described [here](https://docs-cortex.paloaltonetworks.com/r/Cortex-XSIAM/Cortex-XSIAM-Documentation/Broker-VM).
 
-* Pay attention: Timestamp parsing is configured for **mmm dd HH:MM:SS** format in UTC timezone.
-                 This can be done by running the following command on the relevant Linux server:
-                 `sudo timedatectl set-timezone UTC`
+#### Configuration Steps
 
+1. Navigate to **Settings** &rarr; **Configuration** &rarr; **Data Broker** &rarr; **Broker VMs**. 
+2. Go to the **APPS** column under the **Brokers** tab and add the **Syslog** app for the relevant broker instance. If the Syslog app already exists, hover over it and click **Configure**.
+3. Click **Add New**.
+4. When configuring the Syslog Collector, set the following parameters:
+   | Parameter     | Value    
+   | :---          | :---                    
+   | `Vendor`      | Enter **linux**. 
+   | `Product`     | Enter **linux**. 
```

Comment:
Remove the word "Enter" in the value column

Reply:
Fixed:
https://github.com/demisto/content/pull/36882/commits/be339141a34bd645960ec4e017b1ab64e68a7424#diff-547b4347d8e4a399e431acab3ca503579d3d23c0f8854847ee85c97343769646R25